src/common/layers/Layer.js

```diff
@@ -86,15 +86,12 @@ export default class Layer extends Observable {
         writable: true,
       },
       copyright: {
-        get: () => {
-          return this.get('copyright', copyright);
-        },
-        set: (newCopyright) => {
-          return this.set('copyright', newCopyright);
-        },
+        value: copyright,
+        writable: true,
```

Comment:
Leave as it was. I've only leave copyright as property for backward compatibility

Reply:
As it was when integrating those layers in react-spatial, 'layer.copyright' will give undefined.

Reply:
yes you're right . probabyl becaus eit's neve rinitalized . ok to make it writable it's easire.